core/src/main/scala/cr/pulsar/Consumer.scala

```diff
@@ -71,40 +69,40 @@ object Consumer {
   case class Message[A](id: MessageId, key: MessageKey, payload: A)
   case class DecodingFailure(msg: String) extends Exception(msg) with NoStackTrace
 
-  private def mkConsumer[F[_]: Async: Spawn, E: Schema](
+  private def mkConsumer[F[_]: Sync: FutureLift, E: Schema](
```

Comment:
Is `Sync` being used at all?

Reply:
Yeah, by these guys:
```
override def ack(id: MessageId): F[Unit]  = F.delay(c.acknowledge(id))
override def nack(id: MessageId): F[Unit] = F.delay(c.negativeAcknowledge(id))
```

There is no async alternative for `negativeAcknowledge`